### PR TITLE
The great resprite part 1

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

adopted a shit ton of sprites from TG, corvax (which is still TG :trollface:)

## Why / Balance

breathing new life into the game
also they look cool as fuck :godo:

## Media
image
image
~different tiles~ no more

## Breaking changes

we'll need to reject almost all other new/edited sprites from wizden
shouldn't be a huge deal. i think.

**Changelog**

:cl:
- add: The grerat resprite part 1

## Summary by CodeRabbit

- **Documentation Updates**
	- Updated copyright information across multiple texture resources to reflect new commit references and additional contributors.
	- Enhanced attribution for various assets, recognizing contributions from multiple artists.
	- Introduced new metadata structure for a clothing item, including dimensions and state definitions. 

These changes improve the clarity and accuracy of asset attribution, ensuring proper credit is given to all contributors.

### CODE REVIEW

File: Resources/Textures/Clothing/Head/Hardsuits/capspace.rsi/meta.json [hunk 1]
```diff
@@ -14,17 +14,25 @@
       "name": "icon-flash"
     },
     {
-      "name": "off-equipped-HELMET",
+      "name": "equipped-HELMET",
```
[the remark this answers is not in the document]
Comment (whateverusername0) [reply Resources/Textures/Clothing/Head/Hardsuits/capspace.rsi/meta.json:17]: the off and on indicates the light sprites being enabled and disabled. this is intended

**Comment by Piras314** [comment]
conflicts @whateverusername0

**Comment by Verbalase** [comment]
paradise love wya ):

**Comment by Aidenkrz** [comment]
chat let me just re-add a ton of TG sprites after we explicitly removed the TG sprites to replace the TG sprites with Paradise sprites that are now going to be replaced by TG sprites.

Thoughts?

**Comment by Verbalase** [comment]
Yeah, we’re going in a paradise direction bro…

**Comment by whateverusername0** [comment]
> Yeah, we’re going in a paradise direction bro…

paradise also has most of the sprites taken from TG...

**Comment by quri1q** [comment]
rd overcoat when?
labcoat_rd
rdturtle
rdturtle_skirt

**Comment by JoeHammad1844** [comment]
i dont like nu-TG sprites. I do like the brigmedic and ninja sprites but everything else just isn't it

**Comment by JoeHammad1844** [comment]
especially the ID card and ERT sprites just suck

**Comment by whateverusername0** [comment]
i'd say fuck it give it a go (that's why i made the pr in the first place)
but it's up to piras now

**Comment by JoeHammad1844** [comment]
i just remembered, the disposal unit resprite should probably be reverted

**Comment by Piras314** [comment]
for things that haven't already been resprited from paradise I guess tg is better than nothing but for stuff like tiles I prefer paradise (especially empty tiles holy those look horrible)

**Comment by whateverusername0** [comment]
> for things that haven't already been resprited from paradise I guess tg is better than nothing but for stuff like tiles I prefer paradise (especially empty tiles holy those look horrible)

sure

**Comment by quri1q** [comment]
> for things that haven't already been resprited from paradise I guess tg is better than nothing but for stuff like tiles I prefer paradise (especially empty tiles holy those look horrible)

> especially the ID card and ERT sprites just suck

I really like the ids and empty tiles what do yall even think is wrong with them

**Comment by whateverusername0** [comment]
reverted tiles

**Pull request merged**